From a1b17b3cefb92018cfe992aa043a78c9c7a38e0f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 03 Sep 2021 17:53:18 +0300
Subject: [PATCH] [SW-0] ppboot: fix typo in error message

s/oveflowed/overflowed/
No other changes, just a typo in the message.

Change-Id: I93b240e517dd284e4137d653ae6fb94ce2e3875d
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/app/ppboot/ppboot.c b/app/ppboot/ppboot.c
index dbc3cb0..3a3901b 100644
--- a/app/ppboot/ppboot.c
+++ b/app/ppboot/ppboot.c
@@ -245,7 +245,7 @@
 void ppboot_check_stack_monitor(void)
 {
 	if (__stack != STACK_CANARY_MAGIC_WORD) {
-		preboot_puts("The stack has oveflowed\n");
+		preboot_puts("The stack has overflowed\n");
 		preboot_puts_hex("Stack Canary value = ", __stack, false);
 		cpu_boot_err_clrset(CPU_BOOT_ERR0_BOOT_FW_CRIT_ERR, true);
 	}
